### PR TITLE
docs: fix link in auth0.md

### PR DESCRIPTION
**Pull request by kalyan-vurugonda**
The link to the user management overview page is not working, so I replaced it and slightly changed the line.
If you want any more changes please tell me.

Signed-off-by: Kalyan Vurugonda <Vurugondakalyan811@gmail.com>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [x] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

**Comment by kalyan-vurugonda** [comment]
Hi @rbreez, can you look into this PR.

**Comment by kalyan-vurugonda** [comment]
Hi @rbreeze, can you please look into this PR.

**Review by rbreeze — APPROVED** [review]

File: docs/operator-manual/user-management/auth0.md [hunk 1]
```diff
@@ -69,5 +69,5 @@ data:
 
 <br>
 
-!!! note "Storing Client Secrets"
-    Details on storing your clientSecret securely and correctly can be found on the [User Management Overview page](../../user-management/#sensitive-data-and-sso-client-secrets).
\ No newline at end of file
+!!! **Note** : "Storing Client Secrets"
```
Comment (crenshaw-dev) [inline docs/operator-manual/user-management/auth0.md:72]: @kalyan-vurugonda can you revert this line? it breaks the ReadTheDocs markdown formatting.

image

You can test with `make serve-docs-local`

**Review by crenshaw-dev — CHANGES_REQUESTED** [review]

File: docs/operator-manual/user-management/auth0.md (hunk 1 above)
Comment (kalyan-vurugonda) [reply docs/operator-manual/user-management/auth0.md]: Hi @crenshaw-dev
I made changes as you mentioned above.

**Review by crenshaw-dev — APPROVED** [review]
link works in both GitHub and RTD. Thanks @kalyan-vurugonda!

**Review by crenshaw-dev — APPROVED** [review]
lgtm, thanks @kalyan-vurugonda!

Not going to cherry-pick, because I think the issue is only in GitHub, not in RTC.

**Comment by kalyan-vurugonda** [comment]
Thank you @crenshaw-dev

**Pull request merged**